scripts/cleanup.py

```diff
@@ -180,8 +202,8 @@ def main():
 
     args = parser.parse_args()
     application_limits: Dict[str, int] = {}
-    for x in args.limits:
-        parts = x.rsplit(":", 1)
+    for l in args.limits:
+        parts = l.rsplit(":", 1)
```

Comment:
`l` may be a poor name for a var due to the confusion with `I` and `1`, but :man_shrugging: